[zziplib] Build failure for community triplet arm64-android

**Host Environment**
- OS: Windows 10
- Compiler: Android NDK r24 / CLang

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install zziplib:arm64-android`

**Failure logs**
This is the log (note that the dependency zlib built successfully in a previous run of the command)

```
Computing installation plan...
The following packages will be built and installed:
    zziplib[core]:arm64-android -> 0.13.72
Detecting compiler hash for triplet arm64-android...
Restored 0 packages from C:\Users\Michael\AppData\Local\vcpkg\archives in 300.8 us. Use --debug to see more details.
Installing 1/1 zziplib:arm64-android...
Building zziplib[core]:arm64-android...
-- Using community triplet arm64-android. This triplet configuration is not guaranteed to succeed.
-- [COMMUNITY] Loading triplet configuration from: C:\Projekte\Tools\vcpkg\triplets\community\arm64-android.cmake
-- Using cached gdraheim-zziplib-v0.13.72.tar.gz.
-- Cleaning sources at C:/Projekte/Tools/vcpkg/buildtrees/zziplib/src/v0.13.72-92df923ffc.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source C:/Projekte/Tools/vcpkg/downloads/gdraheim-zziplib-v0.13.72.tar.gz
-- Applying patch no-release-postfix.patch
-- Using source at C:/Projekte/Tools/vcpkg/buildtrees/zziplib/src/v0.13.72-92df923ffc.clean
-- Configuring arm64-android
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:83 (message):
    Command failed: C:/Projekte/Tools/vcpkg/downloads/tools/ninja/1.10.2-windows/ninja.exe -v
    Working Directory: C:/Projekte/Tools/vcpkg/buildtrees/zziplib/arm64-android-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\Projekte\Tools\vcpkg\buildtrees\zziplib\config-arm64-android-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:238 (vcpkg_execute_required_process)  
  ports/zziplib/portfile.cmake:13 (vcpkg_cmake_configure)
  scripts/ports.cmake:146 (include)


error: building zziplib:arm64-android failed with: BUILD_FAILED
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+zziplib
You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[zziplib]+Build+error
Include '[zziplib] Build error' in your bug report title, the following version information in your bug description, and attach any relevant failure logs from above.
    vcpkg-tool version: 2022-05-05-67e17c1782801cf481be9ac0b3765dff3e4bdeb8
    vcpkg-scripts version: 8e1f46de1 2022-06-10 (17 hours ago)
```
Attached is the mentioned file
[config-arm64-android-out.log](https://github.com/microsoft/vcpkg/files/8883892/config-arm64-android-out.log)

**Additional context**
I think the issue is with cross-compiling on Windows for Android:

    Could NOT find UnixCommands (missing: BASH CP GZIP MV RM)

The error comes from this line:
https://github.com/gdraheim/zziplib/blob/7f8e01b5fa7f076bbe4f026d6a30160272da69fe/zzip/CMakeLists.txt#L39
and is triggered via:
https://github.com/gdraheim/zziplib/blob/7f8e01b5fa7f076bbe4f026d6a30160272da69fe/zzip/CMakeLists.txt#L22

I think from reading the CMake docs, the `if(UNIX)` line is wrong, as it checks if the cross-compiling target system is a Unix-like system (which Android is), but the unix tools are required on the build host, which is a Windows one in my case. So the fix would be to check for `if(CMAKE_HOST_UNIX)` instead. Do you agree?

--- a/ports/zziplib/portfile.cmake
+++ b/ports/zziplib/portfile.cmake
@@ -11,6 +11,11 @@ vcpkg_from_github(
 string(COMPARE EQUAL VCPKG_CRT_LINKAGE "static" MSVC_STATIC_RUNTIME)
 string(COMPARE EQUAL VCPKG_LIBRARY_LINKAGE "static" BUILD_STATIC_LIBS)
 
+# on Windows hosts, the UnixCommands are not available; disable options that use them
+if(VCPKG_HOST_IS_WINDOWS)
+    set(ZZIPLIB_OPTIONS "-DZZIP_COMPAT=OFF;-DZZIP_PKGCONFIG=OFF")
+endif()
+
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
@@ -24,6 +29,7 @@ vcpkg_cmake_configure(
         -DZZIPBINS=OFF
         -DZZIPTEST=OFF
         -DZZIPDOCS=OFF
+        ${ZZIPLIB_OPTIONS}
 )
 vcpkg_cmake_install()
 

--- a/ports/zziplib/vcpkg.json
+++ b/ports/zziplib/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "zziplib",
   "version": "0.13.72",
-  "port-version": 2,
+  "port-version": 3,
   "description": "library providing read access on ZIP-archives",
   "homepage": "https://github.com/gdraheim/zziplib",
   "license": "LGPL-2.0-or-later OR MPL-1.1",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8166,7 +8166,7 @@
     },
     "zziplib": {
       "baseline": "0.13.72",
-      "port-version": 2
+      "port-version": 3
     }
   }
 }

--- a/versions/z-/zziplib.json
+++ b/versions/z-/zziplib.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "4408845ee5c914b49838db10b1e7dec234ee5981",
+      "version": "0.13.72",
+      "port-version": 3
+    },
+    {
       "git-tree": "ba836047fca40d155c24a986af7cf5283692a4d6",
       "version": "0.13.72",
       "port-version": 2

